CS2113 IP/src/main/java/Duke.java

```diff
@@ -0,0 +1,108 @@
+import java.util.Scanner;
+
+public class Duke {
+    public static void Greet() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println(logo);
+        String horizontalLine = "________________________";
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(horizontalLine);
+    }
+
+    public static void listOperations() {
+        Scanner sc = new Scanner(System.in);
+        String horizontalLine = "________________________";
+        boolean isBye;
+        boolean isList;
+        boolean isDone;
+        boolean isTodo;
+        boolean isDeadline;
+        boolean isEvent;
+
+        Task[] taskList = new Task[100];
+
+        do {
+            String userInput = sc.nextLine();
+            isBye = userInput.equals("bye");
+            isList = userInput.equals("list");
+            isDone = userInput.startsWith("done");
+            isTodo = userInput.startsWith("todo");
+            isDeadline = userInput.startsWith("deadline");
+            isEvent = userInput.startsWith("event");
+            System.out.println(horizontalLine);
+
+            if (isBye) {
+                System.out.println("Bye. Hope to see you again soon!");
+            } else if (isList) {
+                listTask(taskList, Task.taskCount);
+            } else if (isDone) {
+                markTask(taskList, userInput);
+            } else if (isTodo) {
+                addTask(taskList, Task.taskCount, userInput, TaskType.TODO);
+            } else if (isDeadline) {
+                addTask(taskList, Task.taskCount, userInput, TaskType.DEADLINE);
+            } else if (isEvent) {
+                addTask(taskList, Task.taskCount, userInput, TaskType.EVENT);
+            } else {
+                System.out.println("Error input..");
+            }
+            System.out.println(horizontalLine);
+
+        } while (!isBye);
+    }
+
+    private static void addTask(Task[] taskList, int taskCount, String userInput, TaskType specificTask) {
```

Comment:
perhaps you could consider putting this function into another class? because it might be more suitable elsewhere like say the Task class and it'll declutter your main class as well